tests/testthat/testdata/configs/tasks-v5-req-task-id.json

```diff
@@ -0,0 +1,171 @@
+                        "target": {
+                            "required": null,
+                            "optional": ["wk flu hosp rate category"]
+                        },
+                        "horizon": {
+                            "required": null,
+                            "optional": [0, 1, 2, 3]
+                        },
+                        "location": {
+                            "required": [
+                                "US",
+                                "01"
+                            ],
+                            "optional": [
+                                "02",
+                                "04",
+                                "05"
+                            ]
+                        },
+                        "variant": {
+                            "required": null,
+                            "optional": null
+                        },
+                        "target_end_date": {
+                            "required": null,
+                            "optional": [
+                                "2022-10-22", "2022-10-29", "2022-11-05",
+                                "2022-11-12", "2022-11-19", "2022-11-26"
+                            ]
+                        }
+                    },
+                    "output_type": {
+                        "pmf": {
+                            "output_type_id": {
+                                "required": [
+                                    "low",
+                                    "moderate",
+                                    "high",
+                                    "very high"
+                                ]
+                            },
+                            "is_required": true,
+                            "value": {
+                                "type": "double",
+                                "minimum": 0,
+                                "maximum": 1
+                            }
+                        }
+                    },
+                    "target_metadata": [
+                        {
+                            "target_id": "wk flu hosp rate category",
+                            "target_name": "week ahead weekly influenza hospitalization rate category",
+                            "target_units": "rate per 100,000 population",
+                            "target_keys": {
+                                "target": "wk flu hosp rate category"
+                            },
+                            "target_type": "ordinal",
+                            "description": "This target represents a categorical severity level for rate of new hospitalizations per week for the week ending [horizon] weeks after the reference_date, on target_end_date.",
+                            "is_step_ahead": true,
+                            "time_unit": "week"
+                        }
+                    ]
+                },
+                {
+                    "task_ids": {
```

Comment:
Note that in this model task, all task IDs apart from the derived task ID `target_end_date` have a required value. The model task is completely missing however from the parquet test file below though.